src/WebJobs.Extensions.DurableTask/Listener/TaskEntityShim.cs

```diff
@@ -147,6 +148,10 @@ internal void Rehydrate(string serializedInput)
 
         public override async Task<string> Execute(OrchestrationContext innerContext, string serializedInput)
         {
+#if !FUNCTIONS_V1
```

Comment:
In general, it is best to keep the conditional compilation to a minimum, so defining the variable up here and then having to conditional compilation every time we want to set it makes the code harder to read. I would almost prefer we just remove the conditional compilation around defining/setting this variable, and just suppress any warnings we get in Functions V1 for the variable never being referenced.

Reply:
I just realized Chris said either option works for him. There is technically nothing wrong with this over suppressing the warning, but as mentioned, I prefer the suppressing the warning, as it has less impact on readability imo.

Reply:
I think you're right, after trying the alternative, I think this is much cleaner. Fixing it in the next commit!